docs/sources/upgrading/_index.md

```diff
@@ -145,6 +145,22 @@ PR [#3842](https://github.com/grafana/loki/pull/3842)/[#4253](https://github.com
 - `cortex_runtime_config*` -> `loki_runtime_config*`
 - `cortex_chunks_store*` -> `loki_chunks_store*`
 
+#### Fork cortex chunk storage into Loki
+
+PR [#3842](https://github.com/grafana/loki/pull/3842) **cyriltovena**: Fork cortex chunk storage into Loki.
+
+Since Cortex doesn't plan to use `chunk` package anymore, we decided to fork it into our storage package to
+be able to evolve and modify it easily. However, as a side-effect, while Cortex doesn't remove this package from
+its source code, we have duplicated protos being registered, which raises messages similar to the ones below.
+For now, you should just ignore them.
```

Comment:
Out of curiosity, do we have any long-term workaround or solution for it?

Reply:
I think Cortex has plans to remove this package in the future. So, although boring, I think the long-term solution is to wait until they remove it.